### PR TITLE
Use entry points for pyinstaller builds

### PR DESCRIPTION
**Pull request by amolenaar**
### PR Type
What kind of change does this PR introduce?

- [ ] Bug fix
- [ ] Feature
- [x] Chore (refactoring, formatting, local variables, other cleanup)
- [ ] Documentation content changes

### What is the current behavior?

Currenly a Pyinstaller build consists of 3 steps: create `gaphor-script.py`, create a (windows) version file, build the package.

### What is the new behavior?

Packaging is done in one step. The version file step is included in the spec file. Instead of creating a script to import all entry points, use the `collect_entry_point` function from PyInstaller to collect the entry points.

### Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: _packaging/gaphor.spec [hunk 1]
```diff
@@ -70,14 +87,27 @@ a = Analysis(
...
+
 pyz = PYZ(a.pure, a.zipped_data, cipher=block_cipher)
+
+pyinstaller_versionfile.create_versionfile(
+    output_file="windows/file_version_info.txt",
+    version=get_semver_version(),
+    company_name="Gaphor",
+    file_description="Gaphor",
+    internal_name="Gaphor",
+    legal_copyright="Copyright © 2001-2021 Arjan J. Molenaar and Dan Yeaw.",
```
Comment (danyeaw) [inline _packaging/gaphor.spec:106]: While we are at it, what do you think about importing datetime and then:

`legal_copyright=f"Copyright 2011-{datetime.date.today().year}...`

**Review by danyeaw — APPROVED** [review]
Really nice simplification!

File: _packaging/gaphor.spec (hunk 1 above)
Comment (amolenaar) [reply _packaging/gaphor.spec:106]: I want to make one copyright variable: we also need it for the macOS build.

Comment (danyeaw) [reply _packaging/gaphor.spec]: Great, I like it!

**Pull request merged**